lib/Db/Question.php

```diff
@@ -79,8 +79,6 @@ public function getExtraSettings(): array {
 	 * @param object|array $extraSettings
 	 */
 	public function setExtraSettings($extraSettings) {
```

Comment:
```suggestion
	public function setExtraSettings(array $extraSettings) {
```

I think the todo meant this

Reply:
Ah ok, I thought it was about the param description above...

Reply:
No php 8 allowed typings like `public function foo(array|object $bar)` for older PHP versions you needed that comment

Reply:
So should we keep the `array|object` typing? And can we remove the comment, too?